src/org/joml/Rectanglei.java

```diff
@@ -284,6 +284,29 @@ public boolean containsPoint(Vector2ic point) {
         return containsPoint(point.x(), point.y());
     }
 
+    /**
+     * Test whether the point <code>(x, y)</code> lies inside this BlockRegion.
+     *
+     * @param x the x coordinate of the point
+     * @param y the y coordinate of the point
+     * @return <code>true</code> iff the given point lies inside this BlockRegion; <code>false</code> otherwise
+     */
+    public boolean containsPoint(float x, float y) {
+        return x >= this.minX && y >= this.minY && x <= this.maxX && y <= this.maxY;
```

Comment:
I wonder whether the upper bounds should rather be: `x < this.maxX` and `y < this.maxY` as current this is inconsistent with the `int` version of this method, yielding for exact integer coordinates false with the `int` version and true for the `float` version (for exactly float-representable integers).

Reply:
AABBi uses `<=` for the max bound does the edge of a square count as containing a point?

Reply:
Hmm... it seems that making this consistent everywhere and documenting it in the JavaDocs should be the first thing to do. I'm fine with either way, ("upper edge" being inclusive or exclusive). Do you have any preferences that would make your life easier?

Reply:
if its for consistency then inclusive would be better.